tla-io/src/main/scala/at/forsyte/apalache/io/json/jackson/JacksonRep.scala

```diff
@@ -0,0 +1,47 @@
+package at.forsyte.apalache.io.json.jackson
+
+import at.forsyte.apalache.io.json.JsonRepresentation
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import scala.jdk.CollectionConverters._
+
+/**
+ * A JsonRepresentation using Jackson's JsonNode.
+ */
+sealed case class JacksonRep(value: JsonNode) extends JsonRepresentation {
```

Comment:
I wouldn't abbreviate the child if the parent is not

```suggestion
sealed case class JacksonRepresentation(value: JsonNode) extends JsonRepresentation {
```